server/athenian/api/internal/miners/github/pull_request.py

```diff
@@ -1121,6 +1122,7 @@ async def fetch_jira():
                 _issue.acc_id,
                 _issue_epic.key.label("epic"),
                 _issue.project_id,
+                _issue.priority_name,
```

Comment:
I see no easy way to add this conditionally.
As an alternative, we can `if jira.priorities: raise CancelCache()` if postprocessing